Capstone Project- Module 1

annaeliana7 opened the pull request:
### Capstone Project- Module 1

In this project, I created a website for the people of this world. The purpose was to share the salvation of Jesus Christ, the One and Only Messiah.

The website was created using HTML, CSS, and JavaScript to meet the requirements defined within the scope of the Microverse Capstone project. For this purpose, I follow these steps:

-  Create a repository and set up tools.
-  Chose the topic of the website.
-  Download required asset files.
-  Create Homepage and About Page for mobile. 
-  Deploy Project. 
-  Implement mobile menu using JavaScript. 
-  Add media query to make the website responsive for desktop. 
-  Use JavaScript objects to add speakers' data dynamically. 
-  DOM manipulation using JavaScript to load speakers' data on HomePage. 
-  Create a professional README file and PR description.

annaeliana7 committed:
Add README file to project

annaeliana7 committed:
Add nav bar to  to menu

annaeliana7 committed:
Add style to nav bar in css to mobile version

annaeliana7 committed:
Add details to cards and footer

annaeliana7 committed:
Add testimonials in JavaScript

annaeliana7 committed:
Add about page to html about file

annaeliana7 committed:
Refactor CSS code to correct headline and about section

annaeliana7 committed:
Fix linter errors

annaeliana7 committed:
Update README file to include live demo

christianbiring1 commented on README.md:
```diff
@@ -1,31 +1,133 @@
 
-To get a local copy up and running follow these simple example steps.
 ### Prerequisites
+
+In order to run this project you need:
+
+-Any broswer
+
```
- [OPTIONAL]
- [x] Even some knowledge of HTML CSS, and JavaScript might be cool to be mentioned :smile:

annaeliana7 replied:
Thank you Christian for your optional suggestions. My README file under my branch "Capstone-Mod-1" has the languages I used with this projects. 

I also updated my demo link, moving the link from the main branch to the "Capstone-Mod-1" branch 😊

christianbiring1 commented on about.html:
```diff
@@ -0,0 +1,145 @@
+        <div class="header-top">
+            <div class="wrapper">
+                <nav>
+                    <ul class="top-nav">
+                        <li><img src="images/icons/fb-icon.svg"></li>
+                        <li><img src="images/icons/twitter-icon.svg"></li>
```
- [x] For accessibility purposes, You may consider giving your `img` and alternative text that describes what the image is about.

annaeliana7 replied:
Thank you for your suggestion. I will consider this on my next project. 👍

christianbiring1 commented on index.html:
```diff
@@ -0,0 +1,183 @@
+    </header>
+
+    <main>
+        <section id="headline">
+            <div class="wrapper headline-content">
+                <h3>Saving the World!</h3>
+                <h1>A mission to share the salvation of Jesus Christ, the One and Only Messiah.</h1>
+                <div class="headline-description">
+                    <p>With our King, Jesus Christ's return, literally at our door, our mission is to spread salvation in the hope that everyone will believe in Him. "For God so loved the world, that he gave his only begotten Son, that whosoever believeth in him should not perish, but have everlasting life." - John 3:16</p>
+                </div>
+                <div class="headline-footer">
+                    <h2>Upcoming Event - Unlimited Seats (28 ~ 31 March 2023)</h2>
+                    <h3>@Kefi Unearthed - Israel</h3>
+                </div>
+            </div>
+        </section>
+
```
- [ ] Kindly make sure you stick to the design as much as possible. The typos, font, color, and spacing must be similar.
- [ ] You may consider adding some `padding` or `margin` between the elements in this specific section, as well as using the same font provided in the requirement. 
`FONTS` :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203959594-d219bb76-97ad-4ad4-ba92-9793f5fc00e1.png)
`COLOR`
![image](https://user-images.githubusercontent.com/97100091/203960012-64281e3d-1300-43f1-8f4a-74b1629a396e.png)

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203958380-924b978c-2819-43cc-a25a-55a52e3a5193.png)

The original template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203958173-c085221c-4d50-4afc-bf37-56bdba05a65e.png)

annaeliana7 replied:
Thank you for your comment. The font I used was Lato. The project asked us to make it a custom style I desire and I'm really happy with my project and my customized style, can you please allow me to use my design? 🙏

christianbiring1 commented on index.html:
```diff
@@ -0,0 +1,183 @@
+                    </div>
+                </div>
+            </div>
+            <div class="program-btn">
+                <button type="button">Join us and Spread the Love of Jesus</button>
+            </div>
```
- [ ] Kindly consider readjusting this button to match the template provided in Behance
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203961411-1ae94ee0-d26f-4e84-8992-5f7412a4633e.png)

Template provided :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203961749-cd735103-8799-41b8-b12d-d566458a504a.png)

annaeliana7 replied:
Thank you for your comment. I ask that you please don't make me change my customized design. 🙏 I purposely want a button to expand like a banner. I want the button to remain red and I want it to expand across all five cards. The reason for the button to expand to all five cards is because five is a symbolic Biblical number, which means mercy. I would really appreciate you allowing me to keep my customized design. Thank you.

nicupop729 replied:
- [x] Please ensure that this button will match the design, in order for your project to be approved. :+1:

christianbiring1 commented on index.html:
```diff
@@ -0,0 +1,183 @@
+    <footer id="footer">
+        <div class="footer-logo">
+            <img src="images/logo.jpg" alt="logo">
+        </div>
+        <div class="footer-content">
+            <p>2022 Kefi Unearthed. All rights Reserved</p>
+        </div>
+    </footer>
```
- [ ] To make sure this footer section matches the one in the design, You may consider putting space between them.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203962762-92d54421-5a3d-49ed-9de4-a3760d67ae20.png)

The provided template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203962906-27172930-28e8-43d2-93b7-f1f24113b2e1.png)

annaeliana7 replied:
I added space to my project to see how it would turn out. I didn't like it. I would like to please keep my design customized to my preference. 🙏 Please and thank you.

christianbiring1 commented on about.html:
```diff
@@ -0,0 +1,145 @@
+    </header>
+
+    <main>
+        <section id="headline">
+            <div class="wrapper headline-content t-center">
+                <h1>A mission to share the salvation of Jesus Christ, the One and Only Messiah.</h1>
+                <div class="headline-about-description">
+                    <p>With our King, Jesus Christ's return, literally at our door, our mission is to spread salvation in the hope that everyone will believe in Him. "For God so loved the world, that he gave his only begotten Son, that whosoever believeth in him should not perish, but have everlasting life." - John 3:16</p>
+                </div>
+                <div class="headline-about-footer">
+                    <h2>Please get in touch with us for more information at:</h2>
+                    <h3>kefiunearthed@gmail.com</h3>
+                </div>
```
- [ ] Kindly make sure you stick to the design as much as possible. The spacing of your element(padding, margin) and even the look of the page have to be similar to the design.
- [ ] And also, make sure you use the same font-size, family and weight

YOur design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203964122-6e4174b9-120c-4ea8-bfed-f1962a2d3c24.png)

The provided template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203964053-ea00ed4e-187b-4b8e-b09f-21bde413fece.png)

christianbiring1 commented on about.html:
```diff
@@ -0,0 +1,145 @@
+            </div>
+        </section>
+        <section id="logo">
+            <div class="logo-title">
+                <h2>Kefi Unearthed</h2>
+                <hr>
+            </div>
+            <div class="logo-description">
+                <p class="t-center">Our mission is to share the salvation of Jesus Christ, the One and Only Messiah. <br>
+                    "For as the lightning cometh out of the east, and shineth even unto the west; <br>so shall also the coming of the Son of man be." - Matthew 24:27
+                </p>
+            </div>
+            <div class="logo-content">
+                <img src="images/logo.jpg" alt="logo">
+            </div>
+        </section>
```
- [ ] Please consider sticking to the template as much as possible. `The spacing(margin, padding)`, `color` have to be similar to the design
- [ ] Also you may consider reducing the size of your logo and make it matches with the one provided in the design

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203964880-58392839-f99e-4752-a6d7-e3e57a9c0390.png)

The template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203965112-15d7a454-5926-4da0-b2de-728a8837c22e.png)

annaeliana7 replied:
I thank you for your comment. Again, this is my customized option. I would like to keep Kefi Unearthed bolded as it is. Thank you.

christianbiring1 replied:
I was saying it would be nicer to reduce the size of it. @annaeliana7

christianbiring1 commented on about.html:
```diff
@@ -0,0 +1,145 @@
+        </section>
+        <section id="past-events">
+            <div class="events-title">
+                <h2>Past Global Events</h2>
+                <hr>
+                <p class="t-center">Experience our past two events in photos; locations New York, New York, and Jerusalem, Israel.</p>
+            </div>
+            <div class="events-container">
+                <div class="event-card">
+                    <h2>2022</h2>
+                    <h3>New York, New York</h3>
+                </div>
+                <div class="event-card">
+                    <h2>2022</h2>
+                    <h3>Jerusalem, Israel</h3>
+                </div>
+            </div>
```
- [ ] Kindly consider sticking to the design as much as possible. You may consider giving these images that orange background as it is in the design.
- [ ] Also you may consider using the correct color on your element as it is on the provided template.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203966180-167282f6-3b17-4ee9-8e5b-6f86ec0d4b7b.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203966315-cd39cf74-8c09-42c0-89ee-750466c2fb48.png)

annaeliana7 replied:
I'm sorry, I ask that I can please keep my design custom to my preference. I made this webpage with with my customized options and selections. Please allow me to keep my design as is 🙏

christianbiring1 commented on about.html:
```diff
@@ -0,0 +1,145 @@
+                    <div class="icon-menu-mobile">
+                        <img src="images/icons/hamburger-icon.svg" alt="Hamburger icon">
+                    </div>
+                    <ul class="bottom-nav">
+                        <li><a  class="active" href="about.html">About</a>
```
- [ ] When the user is navigating the about page, it would be nicer to have a `link` that will be redirecting them to the home page and not always on the same about page for a better user experience.

annaeliana7 replied:
Thank you for your suggestion. I will consider this on my next project. 👍

christianbiring1 commented on style.css:
```diff
@@ -0,0 +1,690 @@
+  flex-direction: column;
+  gap: 20px;
+}
+
+.headline-content h1 {
+  font-size: 40px;
+  color: var(--red-color);
+  text-transform: uppercase;
+  font-weight: 800;
+}
+
+.headline-description {
+  padding: 16px;
+  border: 2px solid var(--white-color);
+  color: var(--black-color);
```
- [ ] KIndly make sure you stick to the template. On the mobile version of your MVP your font, and spacing are supposed to be similar to the design 

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203968067-1b79a4cc-49d8-45d7-951d-75248409632b.png)

![image](https://user-images.githubusercontent.com/97100091/203968510-2d65d0df-5dd9-4a4c-98d3-d86f12a02d79.png)


The provided template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203968299-7ee75b59-723c-4a03-a918-99e5d457aab7.png)

![image](https://user-images.githubusercontent.com/97100091/203968682-25df1625-4ccf-4df3-866a-1f34cbb31b49.png)

annaeliana7 replied:
We were supposed to customize our page. I ask again that you please allow me to keep my customized version. Thank you.

christianbiring1 commented on style.css:
```diff
@@ -0,0 +1,690 @@
+  margin-top: 24px;
+}
+
+.program-btn button {
+  width: 80%;
+  padding: 24px;
+  color: var(--grey-color);
+  background-color: var(--red-color);
+  font-size: 20px;
+  cursor: pointer;
+  margin-bottom: 60px;
+}
```
- [ ] Kindly, readjust the size and spacing of your Join button to make it matches the provided template.
- [ ] Also you may consider readjusting the color of the elemet.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203970029-1e2f0e77-c4d8-441e-bef0-9dd8301c796c.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203970308-92784576-86dd-4635-aefc-9eed4b9eb7df.png)

annaeliana7 replied:
This is my customized selection. Please allow me to keep my design according to my own customized style I desire.

christianbiring1 reviewed (changes requested):
Hi @annaeliana7 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
![163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856](https://user-images.githubusercontent.com/97100091/203971683-c48ab205-b4d5-406e-8844-1543865a457a.gif)

## Suggested changes
- [ ] `KINDLY CONSIDER READJUSTING THE SPACING BETWEEN YOUR ELEMENT AND AS WELL AS THE COLOR  AND FONT APPLIED TO YOUR TEM`
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Providing a correct `live Demo` link in your README file

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

annaeliana7 committed:
Update about page

annaeliana7 commented:
Deployed page under branch Capstone-Mod-1 to see if deployment error would be resolved. After deploying the branch, my pull request got closed. I reopened my pull request.

annaeliana7 committed:
Update README file

annaeliana7 committed:
Fix error on github with deployment and updated deploy link on README

mckent05 reviewed (changes requested):
Hi @annaeliana7,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![almost there](https://user-images.githubusercontent.com/73607512/204055451-40339bcb-54f1-40a8-b564-0a12ffc95348.gif)



 ## Highlights
- All linter checks are passing ✔️ 
- Your PR is professional 👍 👍 


 ## Required Changes ♻️
- I see that you are yet to implement the changes suggested by the previous reviewer, while it is good that you want to customize your page 👏 , for this milestone you are only allowed to change and customize the content of your page as you have already done 👍 👍 . But it is very important that you stick to the design styling (spacing, margins, fonts, color e.t.c) 🙏 , this is necessary because when you start your job search and recruiters start to view your portfolio or repo 😄, it would give them confidence in your ability to replicate business requirements, giving you an edge over your fellow job searchers 💯. I Plead that you implement the suggested changes given by the previous reviewer so you can have your project approved quickly 👍 👍 
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

nicupop729 commented:
PR under second opinion review.

Please don't push any more commits until I will write here my findings.

nicupop729 reviewed (commented):
Hello @annaeliana7 👋,

_This is an additional review as a result of the second opinion request._

Kudos for the good work on this Capstone! :clap:

Please notice that here at Microverse we are trying to replicate the workflow from a company. When we will work and build a project at our future work company, we will not be able to say to the Project Manager, or to the designer that we like best our own design. As developers, we need to code and create a web app after a design provided for us, and not after a design built by us.

In your free time, I encourage you to build as many web apps as you can, using your own design or replicating designs from other sources. 
But in future works here at Microverse, please make sure to follow the Project Requirements and the advice from Code-Reviewers as close as possible. 

As I appreciate your work and your effort for this Capstone, I only made a mandatory change in your repo. Please find it in the inline comments.

As an optional request, as it was not mentioned by the previous reviewers, please ensure that a user can return to the Home page from the About page in the mobile version. At the moment, in the mobile menu, I could not find a link to the Home page.

After implementing at least that mandatory requirement (I strongly recommend implementing my optional suggestion too), you can ask for another review in the dashboard.

Good luck and Keep rocking :four_leaf_clover:  🚀!

annaeliana7 committed:
Add Home link to mobile menu and Refactor program button on desktop version

annaeliana7 commented:
@nicupop729, thank you for your review and comments! 🙏 
https://github.com/annaeliana7/Capstone-Jesus-Movement/pull/1#pullrequestreview-1194881807

I have made two changes to my project, as you have suggested. 

1. I have added a link to the mobile version to direct people to the Home page
2. I have changed the .program-btn on the desktop version to match the design. 

Thank you for taking the time to review my project and for clarifying your request.

annaeliana7 committed:
Update made to event

christianbiring1 reviewed (approved):
Hi @annaeliana7 , 
# STATUS: APPROVED :tada: 
![nicee](https://user-images.githubusercontent.com/97100091/204102329-7b110f5c-4a32-4990-9f4d-092831b5a905.gif)

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
